setup.py

```diff
@@ -282,8 +282,8 @@ def finalize_options(self):
     'sphinx',
     'sphinxcontrib-napoleon',
     'plotly',
-    'pattern <= 2.6',
```

Comment:
why?

Reply:
I can't find this in PyPI anywhere.

https://pypi.org/search/?q=pattern

Reply:
btw, I'm +1 for fully drop it from gensim at all